Neos.Flow/Classes/Security/Authorization/Privilege/AbstractPrivilege.php

```diff
@@ -74,7 +74,7 @@ public function injectObjectManager(ObjectManagerInterface $objectManager)
      * @param string $permission One of the constants GRANT, DENY or ABSTAIN
      * @param PrivilegeParameterInterface[] $parameters
      */
-    public function __construct(PrivilegeTarget $privilegeTarget, $matcher, $permission, array $parameters)
+    public function __construct(PrivilegeTarget $privilegeTarget, $matcher, string $permission, array $parameters)
```

Comment:
IMO we should either add all type hints (for `$matcher`, too) or none

```suggestion
    public function __construct(PrivilegeTarget $privilegeTarget, $matcher, $permission, array $parameters)
```